adagrams/game.py

```diff
@@ -1,11 +1,169 @@
+'''
+A program to set up and run the Adagrams Game.
+The goal of the game is to create words drawn from a 
+limitted letter pool. Each word is scored based off 
+of which letters were used and based off of length.
+
+Used the random module in order to selet a random array
+of letters from the LETTER_POOL and add the to the
+letter bank.
+
+Used Counter subclass from the collections module 
+to collect the information on the remaining amount 
+of each letter in the letter bank.
+
+'''
+
+
+import random
+from collections import Counter
+
+LETTER_POOL = {
```

Comment:
I like that this defined as a "constant" outside the function. Moving a large chunk of data out from a function helps declutter the function.

And I see what you did there with the letter frequency and letter score in the list. You might even consider renaming LETTER_POOL to something like LETTER_FREQ_WITH_SCORES to indicate that this dict holds 2 different kinds of information.